.env.production.example

```
@@ -0,0 +1,59 @@
+# /data/keys. Generate each secret with:
+#
+#   node -e "console.log(require('crypto').randomBytes(32).toString('hex'))"
+#
+# Hex is not a style preference: these values go into PostgreSQL connection
+# URLs, where a password containing :, /, @, ? or # has to be percent-encoded
+# and silently fails authentication when it is not.
+
+# --- The address members type ----------------------------------------------
+# Invitation and sign-in links are built from this, and the session cookie is
+# issued for it. Use the public https:// address, not localhost, on a server.
+APP_URL="http://localhost:3000"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)